### ISSUE TITLE
Web _ Home Page - Poor Color Contrast of White Text on Colored Images

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#71]
The white text on the red or blue image backgrounds does not have enough color contrast in spots where the image is lighter in color. The white text or focus border and the lighter parts of the blue images have a color contrast ratio of 3.5:1 and need to be 4.5:1 or higher to pass. 

IE 11 uses a white dotted line that is as hard to see as the blue one from Firefox.

**Comment by KyleLivingstonDOL** [comment]
Hi John. To keep consistency with the apps and color palate, we are looking to keep the images, color, and font color the way it is. I can work with our team here to put this on the list to look at addressing in future updates.

**Comment by jfc3-dol** [comment]
Yes, all three platforms need to have enough color contrast to be able to be read and should be consistent because people might use a phone or a tablet and then their computer or the other way around.

**Comment by trevitalgit** [comment]
Please compare before and after.
![image](https://user-images.githubusercontent.com/53573580/63095581-9aa42300-bf39-11e9-8a19-a4b39364db02.png)

**Comment by jfc3-dol** [comment]
Making the color over the images more saturated allows users to the white text easier, so I'm going to close the issue.
